src/main/java/com/google/roomies/data/Listing.java

```diff
@@ -276,9 +294,29 @@ public Builder setListingPrice(String listingPrice) throws UnknownCurrencyExcept
         .setSharedPrice(listingData.get(SHARED_ROOM_PRICE).toString())
         .setSinglePrice(listingData.get(SINGLE_ROOM_PRICE).toString())
         .setListingPrice(listingData.get(LISTING_PRICE).toString())
+        .setComments(getAllCommentsFromCollection(document.getId()))
         .build());
     }
 
+  /**
+  * Gets all comments from a given listing.
+  * 
+  * @return list of comment instances
+  */
+  private static ImmutableList<Comment> getAllCommentsFromCollection(String listingId)
```

Comment:
This should be getAllCommentsForListing and may be rename getAllCommentsInListing which returns documents to 
getAllCommentDocumentsForListing.

Both the comments say same thing for these methods.

Reply:
Also, I don't see any test for Listing class.

Reply:
Renamed. Added tests for listing class and comments class.